src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const operate = (callback, num) => {
+    result = callback(result, num);
+
+    return calculator;
+  };
+
+  const add = (prevResult, num) => {
+    return prevResult + num;
+  };
+
+  const subtract = (prevResult, num) => {
+    return prevResult - num;
+  };
+
+  const multiply = (prevResult, num) => {
+    return prevResult * num;
+  };
+
+  const divide = (prevResult, num) => {
+    return prevResult / num;
+  };
+
+  const reset = () => {
+    result = 0;
+
+    return calculator;
```

Comment:
[CODE KNOWLEDGE] - When creating a method in the object, you don't need to use the `function` keyword, use the shortcut instead. Declare `reset` using the shorthand method syntax.